texar/torch/modules/encoders/t5_encoder.py

```diff
@@ -168,6 +154,7 @@ def default_hparams():
             'dim': dim,
             'embedding_dropout': 0.1,
             'residual_dropout': 0.1,
+            'use_bert_config': False,
```

Comment:
Instead of calling this `use_bert_config`, rather use something like `pre_layer_norm` that describes what the flag actually does.

Reply:
This is what all `TransformerEncoder`s  have been using. I don't think this is an option here. Anything else and it'll raise a param not found error when it goes inside `TransformerEncoder`'s init call. I agree having `use_bert_config` is misleading but I think that's a separate issue that needs to be tracked elsewhere.

Reply:
I agree. Let's open a new issue instead.